README.md

```diff
@@ -19,6 +19,12 @@ It can either be used as a Drupal module or as a standalone Twig extension.
     {% render '@atoms/button.twig' with { name: 'submit' } %}
 
     {% render '@atoms/button--primary.twig' with { label: 'Save' } %}
+
+    {% render [
+      '@pages/section-foo.twig',
+      '@pages/section-bar.twig',
+      '@pages/section-baz.twig',
+    ] %}
```

Comment:
The example could be more clear about the fact that the first existing will be chosen; e.g.:
```diff
     {% render [
-      '@pages/section-foo.twig',
-      '@pages/section-bar.twig',
-      '@pages/section-baz.twig',
+      '@pages/section--' ~ trythis ~ '.twig',
+      '@pages/section--' ~ alternative ~ '.twig',
+      '@pages/section.twig',
     ] %}
```